PUBLISHING-VMWARE.md

```diff
@@ -0,0 +1,65 @@
+# Publishing a Bottlerocket OVA on VMware
+
+This guide will walk through some VMware specific details around making your OVA available as a VM or VM template in one or more software defined datacenters.
+
+### Configuration details
+
+As mentioned in the [PUBLISHING](PUBLISHING.md) guide, the process uses a configuration file called `Infra.toml`.
+For VMware, you can specify details about your various vSphere instances and datacenters in `Infra.toml`, as well as configuration that may be common between datacenters.
+
+It's important to note that we use [`govc`](https://github.com/vmware/govmomi/tree/master/govc) under the hood for interactions with vSphere, so at runtime **all datacenter configuration in `Infra.toml` is overridden by `GOVC_` environment variables.**
```

Comment:
This should probably also explain that it needs to be installed. Reading this one might think that you are using it as a library.

Reply:
We don't need to install it on the host since we're running it in the SDK container.  But it's probably a good note to add anyway.

Reply:
Oh, right.